e2e/cypress/test_module/cypress/support/step_definitions/cart-page.steps.ts

```diff
@@ -0,0 +1,82 @@
+/// <reference types="cypress" />
+
+import { Given, When, Then } from "@badeball/cypress-cucumber-preprocessor";
+import { httpMethod } from "@cypress-e2e/fixtures/global-data";
+
+Given("As a user I am signed in", () => {
+  cy.signIn();
+});
+
+When("I click on the Cart button", () => {
+  cy.get("[data-cy='header-cart-button']").click();

```

Comment:
We have custom commsnd getById. You could use it instead `cy.get("[data-cy='header-cart-button']")`
![image](https://github.com/user-attachments/assets/c2fd2319-248f-45c2-b40d-7b121a2900f2)

Reply:
> We have custom commsnd getById. You could use it instead `cy.get("[data-cy='header-cart-button']")` ![image](https://private-user-images.githubusercontent.com/127191734/351832204-c2fd2319-248f-45c2-b40d-7b121a2900f2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE4NDkwMTQsIm5iZiI6MTcyMTg0ODcxNCwicGF0aCI6Ii8xMjcxOTE3MzQvMzUxODMyMjA0LWMyZmQyMzE5LTI0OGYtNDVjMi1iNDBkLTdiMTIxYTI5MDBmMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyNFQxOTE4MzRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMmJhNWU2ZTc0YWY4YTY5YjkxMDg2NWVkNzliMmUwNGI3MzIxYmZmMjFmYzU3OTBmNWU5MzgzZWU4ODNiMGZlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.cWs-_UQJWwrGnG_nel_ePjPjRxsQsjPcxGR7nF8A41Y)

Rewrited. Thanks